api/tests/routes/pro/get_offerer_members_test.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import pcapi.core.offerers.factories as offerers_factories
+from pcapi.core.testing import override_features
+import pcapi.core.users.factories as users_factories
+from pcapi.models.validation_status_mixin import ValidationStatus
+
+from tests.conftest import TestClient
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns200Test:
+    @override_features(WIP_ENABLE_NEW_USER_OFFERER_LINK=True)
+    def test_get_offerer_members_by_pro(self, app):
+        pro = users_factories.ProFactory(email="offerer@example.com")
+        offerer = offerers_factories.OffererFactory()
+        offerers_factories.UserOffererFactory(user=pro, offerer=offerer)
+        offerers_factories.UserOffererFactory(offerer=offerer, user__email="member.pro@example.com")
+        offerers_factories.UserOffererFactory(
```

Comment:
```suggestion
        offerers_factories.NotValidatedUserOffererFactory(
```
pour garder la cohérence avec la sitution réelle : ainsi l'utilisateur est `NON_ATTACHED_PRO`.
Pareil pour le suivant.